sops/__init__.py

```diff
@@ -95,6 +95,8 @@
 
 INPUT_VERSION = VERSION
 
+DEFAULT_UNENCRYPTED_SUFFIX = '_unencrypted'
```

Comment:
But this is a `DEFAULT_` in the sense of `DEFAULT_TEXT` and `DEFAULT_JSON`, isn't it?